db/migrate/20220125100705_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :posts_counter
+
+      t.timestamps
```

Comment:
- [ ]  Look carefully in your DB folder you have two `create_users.rb` files please remove the other file.

Reply:
Yeah Sure, I'm going to remove it, 
Thank you for the review 🙏🏻